docs/static/netflow-module.asciidoc

```diff
@@ -0,0 +1,157 @@
+for ingestion.
++
+The `--setup` option creates a `netflow-*` index pattern in Elasticsearch and
+imports Kibana dashboards and visualizations. Running `--setup` is a one-time
+setup step. Omit this option for subsequent runs of the module to avoid
+overwriting existing Kibana dashboards.
++
+The command shown here assumes that you're running Elasticsearch and Kibana on
+your localhost. If you're not, you need to specify additional connection
+options. See <<configuring-netflow>>.
+
+. Explore your data in Kibana:
+.. Open your browser and navigate to
+http://localhost:5601[http://localhost:5601]. If security is enabled, you'll
+need to specify the Kibana username and password that you used when you set up
+security.
+.. Open *Netflow: Network Overview Dashboard*.
+.. See <<exploring-data-netflow>> for additional details on data exploration.
+
+[[exploring-data-netflow]]
+==== Exploring Your Data
+
+Once the Logstash Netflow module starts processing events, you can immediately
+begin using the packaged Kibana dashboards to explore and visualize your
+network flow data. 
+
+You can use the dashboards as-is, or tailor them to work better with existing
+use cases and business requirements.
+
+//REVIEWERS: I haven't followed the example of the ArcSight module here (listing all the dashboards) because doing so is a maintainability issue. I think it's better to highlight the important dashboards and let users do some exploration on their own.
+
+[[network-dashboards-netflow]]
+===== Example Dashboards
+
+On the *Overview* dashboard, you can see a summary of basic traffic data and set
+up filters before you drill down to gain deeper insight into the data.
+
+[role="screenshot"]
+image::static/images/netflow-overview.png[Netflow overview dashboard]
+
+For example, on the *Conversation Partners* dashboard, you can see the source
+and destination addresses of the client and server in any conversation.
+
+[role="screenshot"]
+image::static/images/netflow-conversation-partners.png[Netflow conversation partners dashboard]
+
+On the *Traffic Analysis* dashboard, you can identify high volume conversations
+by viewing the traffic volume in bytes.
+
+[role="screenshot"]
+image::static/images/netflow-traffic-analysis.png[Netflow traffic analysis dashboard]
+
+Then you can go to the *Geo Location* dashboard where you can visualize the
+location of destinations and sources on a heat map: 
+
+[role="screenshot"]
+image::static/images/netflow-geo-location.png[Netflow geo location dashboard]
+
+
+[[configuring-netflow]]
+==== Configuring the Module
+
+You can further refine the behavior of the Logstash Netflow module by specifying
+settings in the `logstash.yml` settings file, or overriding settings at the
+command line. 
+
+For example, the following configuration in the `settings.yml` file sets
+Logstash to listen on port 3555 for network traffic data: 
+[source,yaml]
+-----
+modules:
+  - name: netflow
+    var.input.udp.port: 3555
+-----
+
```

Comment:
In this example let's use port 9996 as this is the most commonly used alternative to the default of 2055.